public/data/Horlakz.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "Olakotan Daniel",
+  "type": "personal",
+  "bio": "Front-End Developer",
+  "avatar": "https://github.com/Horlakz.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Horlakz",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Horlakzz",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/Horlakz",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Email",
+      "url": "mailto:horlakz@protonmail.com",
+      "icon": "envelope"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started learning Programming",
+      "date": "March 2016",
+      "icon": "dev.to",
+      "color": "grey",
+      "description": "Started learning programming",
+      "url": "https://www.horlakz.com/"
+    },
+    {
+      "title": "Joined Github",
```

You might want to capitalize the `h` here. What do you think?
```suggestion
      "title": "Joined GitHub",
```